testing.py

```diff
@@ -95,8 +95,6 @@ def setup(self):
         self.longitude_counter = -90
 
     def teardown(self):
-        self._db.rollback()
-
```

Comment:
This rollback was only needed in a single test -- one that causes a database integrity error -- and it seemed weird to roll back the entire database session and then roll back the transaction. It didn't solve the problem I was having but I decided to keep the fix.